services/py-api/src/database/repository/participants_repository.py

```diff
@@ -57,3 +57,9 @@ async def create(
         except Exception as e:
             LOG.exception("Participant insertion failed due to err {}".format(e))
             return Err(e)
+    
+    async def get_verified_random_participants_count(self) -> int:
+        """Returns the count of verified participants who are not assigned to any team."""
+        query = {"verified": True, "team_id": None}
+        count = await self._collection.count_documents(query)
+        return count
```

Comment:
```python
  return await self._collection.count_documents({"email_verified": True, "team_id": None})
```
We don't have a `verified` field in the db model:
<img width="462" alt="image" src="https://github.com/user-attachments/assets/b02b0481-3902-4976-9e87-b46861f3704f">

Reply:
Fixed f33e8f7a85dbefbcda9a089a8ba5a1d3e7e93e9b